src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,22 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+
+    @OneToMany
```

Comment:
```suggestion
    @OneToMany
```
can one smile be used in many comments?

Reply:
Such a condition. Change it if it is considered an error.
![1](https://user-images.githubusercontent.com/79798035/120615128-9f183380-c460-11eb-8c4c-98142f7b8f1e.png)

Reply:
Done.